must-gather/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
I think when we use 1 Makefile only we don't need this

Reply:
see above

Reply:
done